lib/fast_jsonapi/relationship.rb

```diff
@@ -34,12 +34,12 @@ def initialize(
       @lazy_load_data = lazy_load_data
     end
 
-    def serialize(record, serialization_params, output_hash)
+    def serialize(record, included, serialization_params, output_hash)
```

Comment:
This seems like a breaking change.  New parameters should generally be appended to the list and included a default.